databases/week2/Part1_working_task.sql

```diff
@@ -0,0 +1,19 @@
+-- Part 1: Working with tasks
+-- Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+-- task
+insert into task 
+(title, description, created, updated, due_date, status_id, user_id) 
+ values ('homework' , 'Week 2 homework', NOW(), '2020-10-28 03:09:06','2021-09-28 03:09:06',1,1);
+ 
+-- Change the title of a task and due date also Mark a task as complete .
+UPDATE task 
```

Comment:
This was supposed to be 4 queries, not just one :)

Reply:
Yes I forgot to include it in it but I am doing it now

Reply:
The new queries are correct, thank you!

But this one is only supposed to change the title of the task, and nothing else. Can you change it please?

Reply:
Thanks, I changed it

Reply:
Very good, thank you :)